Yoast/wordpress-seo
# Add additional descriptions to the installation wizard

## Summary

This PR can be summarized in the following changelog entry:

* Added additional explanations to the installation wizard.

## Relevant technical choices:

* Added new HTML components where possible, otherwise used the new description property.

## Test instructions

This PR can be tested by following these steps:

* Run `yarn upgrade yoast-components && grunt build` to ensure you have the latest version of `yoast-components`.
* On the first step the two boxes will look weird, this issue was fixed after this branch was created and thus will not show yet ( see #7823 ).
* Open the installation wizard.
* For each step check if the additional description is as specified in the related issue.

Fixes #6933.
